workshop/content/030_basic_content/075_serverless/_index.md

```diff
@@ -5,7 +5,264 @@ weight: 75
+        "FaultParameterValue": "{
+            \"is_enabled\":true,
+            \"fault_type\":\"status_code\",
+            \"delay\":400,
+            \"error_code\":404,
+            \"exception_msg\":\"Fault injected by chaos-lambda\",
+            \"rate\":1
+        }"
+    }
+    ```
+  * "maxDuration": `1` "Minutes"
+
+* "Name": `S04_Wait2`
+  * "Action type": `aws:fis:wait`
+  * "Start after": `S03_EnableStatusCode`
+  * duration: `1` "Minutes"
+
+* "Name": `S05_EnableException`
+  * "Action type": `aws:ssm:start-automation-execution`
+  * "Start after": `S04_Wait2`
+  * "documentArn": the ARN found in the "Outputs" tab of the FisStackServerless [**CloudFormation**](https://console.aws.amazon.com/cloudformation/home?#/stacks/outputs?filteringStatus=active&filteringText=&viewNested=true&hideStacks=false&stackId=FisStackServerless). 
+  * "documentParameters": Reformatted here for legibiity. For the `AutomationAssumeRole` you will need to insert the ARN of the `FisWorkshopLambdaSsmRole` either from the "Outputs" of the cloudformation stack or from the "Prerequisites" section.
+    ```json
+    {
+        "AutomationAssumeRole": "arn:aws:iam::ACCOUNT_ID:role/FisStackServerless-FisWorkshopLambdaSsmRole-xxxxyyyyzzzz",   
+        "FaultParameterValue": "{
+            \"is_enabled\":true,
+            \"fault_type\":\"exception\",
+            \"delay\":400,
+            \"error_code\":404,
+            \"exception_msg\":\"Fault injected by chaos-lambda\",
+            \"rate\":1
+        }"
+    }
+    ```
+  * "maxDuration": `1` "Minutes"
+
+* "Name": `S06_Wait3`
+  * "Action type": `aws:fis:wait`
+  * "Start after": `S05_EnableException`
+  * duration: `1` "Minutes"
+
+* "Name": `S07_DisableFaults`
+  * "Action type": `aws:ssm:start-automation-execution`
+  * "Start after": `S06_Wait3`
+  * "documentArn": the ARN found in the "Outputs" tab of the FisStackServerless [**CloudFormation**](https://console.aws.amazon.com/cloudformation/home?#/stacks/outputs?filteringStatus=active&filteringText=&viewNested=true&hideStacks=false&stackId=FisStackServerless). 
+  * "documentParameters": Reformatted here for legibiity. For the `AutomationAssumeRole` you will need to insert the ARN of the `FisWorkshopLambdaSsmRole` either from the "Outputs" of the cloudformation stack or from the "Prerequisites" section.
+    ```json
+    {
+        "AutomationAssumeRole": "arn:aws:iam::ACCOUNT_ID:role/FisStackServerless-FisWorkshopLambdaSsmRole-xxxxyyyyzzzz",   
+        "FaultParameterValue": "{
+            \"is_enabled\":false,
+            \"fault_type\":\"exception\",
+            \"delay\":400,
+            \"error_code\":404,
+            \"exception_msg\":\"Fault injected by chaos-lambda\",
+            \"rate\":1
+        }"
+    }
+    ```
+  * "maxDuration": `1` "Minutes"
+
+### Target selection
+
+Because we are exclusively using SSM Automation documents we don't need to specify any targets.
```

Comment:
Because we are exclusively using SSM Automation documents  we -> Since we are exclusively using SSM Automation documents, we